src/layout/formPage/SummaryForm/SummaryForm.tsx

```diff
@@ -67,8 +69,10 @@ const SummaryForm:React.FC<IProps> = ({products, checkValue, itemsValue, shipVal
                     shipValue={shipValue}
                     disabledCode={disabledCode}
                 />
+                <Link to="/" className="summaryform button_check">Go back</Link>
             </div>
-            <button onClick={toggleOnClick}>Click me</button>
+            </div>
+            <div></div>
```

Comment:
why empty div?

Reply:
Because of the "sticky" (css position-property value). It wont work if there's only one div.